### PR TITLE
Add returnSearchData function to default story

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Add returnSearchData function to default story

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

- View `/?path=/story/searchandfilter--default-story`
- Should load without error

### CODE REVIEW

File: src/components/SearchAndFilter/SearchAndFilter.stories.mdx [hunk 1]
```diff
@@ -4,7 +4,16 @@ import SearchAndFilter from "./SearchAndFilter";
 
 <Meta title="SearchAndFilter" component={SearchAndFilter} />
 
-export const Template = (args) => <SearchAndFilter {...args} />;
+export const Template = (args) => (
+  <SearchAndFilter
+    {...args}
+    returnSearchData={(searchData) => {
```
Comment (bartaz) [inline src/components/SearchAndFilter/SearchAndFilter.stories.mdx:10]: If the `returnSearchData` prop is required, it should be marked required in the component.
If it's not required, component should render without it.

Comment (barrymcgee) [reply src/components/SearchAndFilter/SearchAndFilter.stories.mdx]: Good point, I've now cleaned that up.

**Comment by bartaz** [comment]
Not sure if we have any control over it, but the example in storybook appears in small scrollable frame:

Screenshot 2020-09-17 at 16 26 52

**Review by bartaz — APPROVED** [review]
LGTM, thanks!

File: src/components/SearchAndFilter/SearchAndFilter.js [hunk 2]
```diff
@@ -212,7 +208,7 @@ const SearchAndFilter = ({
         })}
         <SearchBox
           autocomplete="off"
-          externallyControlled={externallyControlled}
+          externallyControlled={true}
```
Comment (squidsoup) [inline src/components/SearchAndFilter/SearchAndFilter.js:211]: Small thing, but the convention for `true` props, is just to list the prop, e.g.

`<Foo aTrueProp />`

**Pull request merged**